data/json/effects_on_condition/scenario_specific_eocs.json

```diff
@@ -68,7 +68,8 @@
       { "run_eocs": "EOC_SPAWN_MANSION_MONSTERS" },
       {
         "u_run_monster_eocs": [ { "id": "EOC_BANISH_MONSTERS_AROUND_PLAYER", "effect": { "run_eocs": "EOC_BANISH_SELF" } } ],
-        "monster_range": 6
+        "mtype_ids": [ "mon_zombie", "mon_feral_fancy_rapier", "mon_feral_fancy_rapier_fake", "mon_feral_maid_knife" ],
```

Comment:
Now anything that's not on this list can still start adjacent to the player, what's the purpose of specifying them?

Reply:
The 2 safe zones in the mansion have custom monster spawns defined for them, the most lethal monsters seem ok to be removed, but the weakest of them (Zombie children in the panic room, sleek zombies in the sex lair) serve as an immediate threat that the player can fight against and win, useful for story telling. In the panic room there is even a chance of spawning with a still living dog, which was previously removed if you started there.

And limiting the scope of the removal to certain monsters allows me to expand the range of the EOC so no lethal monsters appear just outside the door of your safe zone, but allowing some maids to be around, enemies against whom you can actually plan against this early in your run.

I avoided adding the armored ferals to the list since they are the "boss" of each level, and removing them feel like making it too easy, but maybe a simple teleport could work in their case.

Reply:
GROUP_PANICROOM contains plenty of other lethal monsters though? Notably rottweilers and regular ferals

Reply:
You could change it to remove everything and then spawn less lethal stuff in proximity to you as an alternative

Reply:
I don't think "large" dogs are safe if you get close to them but I could be wrong

Reply:
Huh fair enough, thanks for testing